src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,34 @@
+// Issue #70, Originally from #55
+// Require CC0 waiver 
+// Creator: SamWilsn
+
+// to be included when merged in /src/modules/assertions
+
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+// BOT will fail if no CC0 detected in PR body
+// Just after: Verify correct environment and request context in main.ts, if no CC0 is detected, 
+// BOT will fail and comment: "error: No CC0 waiver detected" 
+
+export const  checkCC0 = () => {
+        
+	const github = getOctokit(GITHUB_TOKEN).rest;
+	if !(github.pulls.get(context.payload?.pull_request?.body).search("CC0")) {
```

Comment:
This should search for the *exact* expected line, not just the string `CC0`.

This should also verify it is the entire line, extra space or text before or after are not allowed.

It should also verify it is the *last* line in the file, perhaps with an optional single newline after it.

Reply:
is this the correct line:

Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

Reply:
is this the correct line:

"Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/)."

(without quotes)

Reply:
Ok